Dockerfile

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.12.9 as builder
+ARG golang_version=golang:1.12.10
```

Comment:
Just out of curiosity, why make this an ARG? Do we want to encourage users trying different golang versions for CAPV?

Reply:
It's more for us and our testing. If you end up keeping it, please make the arg ALL CAPS. Please see [this Dockerfile](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/hack/images/ci/Dockerfile) for an example.

Reply:
@andrewsykim it is for downstream release process to define golang version without need to change upstream.

Reply:
@akutz done, thx